api/package.json

```diff
@@ -62,7 +62,7 @@
     "rxjs": "^7.3.0",
     "strings-file": "^0.0.5",
     "swagger-ui-express": "^4.1.6",
-    "typeorm": "^0.2.37",    
```

Comment:
@bvenceslas I don't see any changes in package.json, can you please remove that from PR? Thank you!

Reply:
will do

Reply:
Hm, seems file is still there in PR :(